test/e2e-tests-latest-eventing.sh

```diff
@@ -95,12 +95,14 @@ function generate_latest_eventing_manifest() {
 # Skip installing istio as an add-on
 initialize $@ --skip-istio-addon
 
+TIMEOUT=20m
+
 # If we got this far, the operator installed Knative Eventing
 header "Running tests for Knative Eventing Operator"
 failed=0
 
-# Run the integration tests
-go_test_e2e -timeout=20m ./test/e2e || failed=1
+# Run the postupgrade tests
+go_test_e2e -tags=postupgrade -timeout=${TIMEOUT} ./test/upgrade || failed=1
```

Comment:
+1 on this. Also good that we're changing the name of this file with https://github.com/knative/eventing-operator/pull/96